tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -113,43 +116,23 @@ public function testBucketDetail()
         $this->_client->dropBucket($bucket['id']);
     }
 
-    /**
-     * @return int
-     */
-    private function getLastEventId()
-    {
-        $lastEvents = $this->_client->listEvents(['limit' => 1]);
-        if (!$lastEvents || !is_array($lastEvents)) {
-            $this->fail('Cannot get last event ID.');
-        }
-        $lastEventId = $lastEvents[0]['id'];
-        $this->assertIsInt($lastEventId);
-        return $lastEventId;
-    }
-
     public function testBucketEvents()
     {
-        $lastEventId = $this->getLastEventId();
+        $this->initEvents($this->_client);
 
-        $description = 'testBucketEvents';
-        $bucketId = $this->initEmptyBucket($this->getTestBucketName($description), self::STAGE_IN, $description);
-        $this->assertIsString($bucketId);
+        // create bucket event
+        $this->_client->listTables($this->getTestBucketId());
 
         // create dummy event
         $event = new Event();
         $event->setComponent('dummy')
             ->setMessage('bucket sample event');
-        $event = $this->createAndWaitForEvent($event);
+        $this->createAndWaitForEvent($event);
 
         // check bucket events
-        $events = $this->_client->listBucketEvents($bucketId, ['sinceId' => $lastEventId]);
+        $events = $this->_client->listBucketEvents($this->getTestBucketId(), ['sinceId' => $this->lastEventId]);
         $this->assertIsArray($events);
         $this->assertCount(1, (array) $events);
-
-        // check dummy event is not among bucket events
-        $this->assertArrayNotHasKey($event['id'], (array) $events);
-
-        $this->_client->dropBucket((string) $bucketId);
```

Comment:
Nechal bych tady assert na to, že je tam nějaká ta konkrétní bucket.detaillisted  eventa. Prostě, že ta jedna je takhle co má být.

Reply:
Pridal som to tam :)